hail/src/main/scala/is/hail/expr/ir/TableIR.scala

```diff
@@ -311,7 +311,13 @@ case class TableParallelize(rowsAndGlobal: IR, nPartitions: Option[Int] = None)
 
   protected[ir] override def execute(ctx: ExecuteContext): TableValue = {
     val hc = HailContext.get
-    val Row(_rows: IndexedSeq[_], globals: Row) = CompileAndEvaluate[Row](ctx, rowsAndGlobal, optimize = false)
+    val (ptype, res) = CompileAndEvaluate._apply(ctx, rowsAndGlobal, optimize = false) match {
+      case Left(()) => (PVoid, ().asInstanceOf[Row])
```

Comment:
This will always fail:

```
scala> ().asInstanceOf[Row]
java.lang.ClassCastException: scala.runtime.BoxedUnit cannot be cast to org.apache.spark.sql.Row
  ... 49 elided
```

Reply:
yeah, you don't need the Left() case.